action/rlp_tx.go

```diff
@@ -138,6 +136,12 @@ func ExtractTypeSigPubkey(tx *types.Transaction) (iotextypes.Encoding, []byte, c
 			encoding = iotextypes.Encoding_ETHEREUM_UNPROTECTED
 			signer = types.HomesteadSigner{}
 		}
+	case types.AccessListTxType:
+		// AL txs are defined to use 0 and 1 as their recovery
+		// id, add 27 to become equivalent to unprotected Homestead signatures.
+		V = new(big.Int).Add(V, big.NewInt(27))
```

Comment:
What if do not add the `27`?

Reply:
this is to follow how Ethereum processes the signature, you can check code of `eip2930Signer` in go-ethereum